homeassistant/components/upcloud/binary_sensor.py

```diff
@@ -29,3 +33,5 @@ async def async_setup_entry(
 
 class UpCloudBinarySensor(UpCloudServerEntity, BinarySensorEntity):
     """Representation of an UpCloud server sensor."""
+
+    _attr_device_class = BinarySensorDeviceClass.POWER
```

Comment:
I don't know this integration, is this correct?

Reply:
Previously it used a hard coded string in its parent class to "power" for all iets platforms... 

So this makes it specific on this platform to fix that (and make mypy happy as it now has catches this bug)